source/index.html.md

```diff
@@ -333,38 +333,53 @@ Example response (snippet):
     },
     "processed_at": "2018-07-09T12:58:56+00:00",
     "amount": 50000,
-    "currency": "EUR",
-    "_embedded": { "card": { "id": "58dabba0-e9ea-4133-8c38-bfa1028c1ed2" } }
+    "currency": "EUR"
 }
 ````
 
 
-## Recurring payments
+## Series of transactions
 
-Recurring payments enable you to repeatedly charge cardholders without having
-them to provide card information for subsequent payments.
+Subscriptions are supported using `series[]`. There are 3 types of
+subscriptions: `installment`, `recurring`, and `unscheduled`
+(merchant-initiated) credential on file.
```

Comment:
Perhaps introduce the UCOF abbreviation here?

Reply:
CoF or COF, and thus, UCoF or UCOF?

Reply:
A few searches suggest that Visa uses C**o**F for Credential-on-File and UC**O**F for "Unscheduled Credential on File".